ci-operator/config/rh-ecosystem-edge/kernel-module-management/rh-ecosystem-edge-kernel-module-management-main.yaml

```diff
@@ -59,6 +67,7 @@ operator:
 promotion:
   excluded_images:
   - ubi8-minimal-entitled
+  - ubi9-minimal-entitled
```

Comment:
Why don't we remove ubi8 completely?

Reply:
Because if we do that, the rehearse jobs don't pass 🙂

Reply:
What is the error we are getting? It may be because we are subtituing registry.access.redhat.com/ubi8/ubi-minimal:8.7 in the signimage with `ubi8-minimal-entitled`?

Should we close https://github.com/rh-ecosystem-edge/kernel-module-management/pull/577 and replace it with a bump to ubi9 and then replace ubi8 with ubi9 in the `main` branch in this PR?

Reply:
Just saw https://github.com/openshift/release/pull/39411#issuecomment-1550963678. I think we should first merge https://github.com/rh-ecosystem-edge/kernel-module-management/pull/579 then try to replace ubi8 with ubi9.

If the rehearsal fails it means we forgot something in KMM. WDYT?

Reply:
> It may be because we are subtituing registry.access.redhat.com/ubi8/ubi-minimal:8.7 in the signimage with ubi8-minimal-entitled?

Exactly, so we need this replacement to still be here in this PR for the rehearsals to pass.

> Should we close rh-ecosystem-edge/kernel-module-management#577 and replace it with a bump to ubi9 and then replace ubi8 with ubi9 in the main branch in this PR?

577 will be closed automatically when 579 is merged. Either way we need to replace both ubi8 and ubi9 for some time to allow the KMM PR to pass.